### PR TITLE
feat : toast

### PR DESCRIPTION
**Pull request by mikesch803**
1. added toast while adding product to wishlist, cart, removing product from cart, wishlist, signin, and adding qty, and removing qty

# files to check 

- [ ] src/components/toast/Toast.jsx
- [ ] src/context/cart-context.jsx
- [ ] src/context/toast-context.jsx
- [ ] src/context/wishlist-context.jsx
- [ ] src/pages/cart-page/Cart.jsx
- [ ] src/pages/landing-page/LandingPage.jsx
- [ ] src/pages/product-page/Product.jsx
- [ ] src/pages/wishlist-page/Wishlist.jsx

## [live link](https://agitated-tesla-b2afa9.netlify.app)

### CODE REVIEW

**Review by arjun-computer-geek — APPROVED** [review]
Looks good. Nice work

File: src/context/wishlist-context.jsx [hunk 1]
```diff
@@ -52,7 +64,13 @@ const WishlistProvider = ({ children }) => {
 
   return (
     <WishlistContext.Provider
-      value={{ wishlistDataHandler, wishlistData, setWishlistData }}
+      value={{
+        wishlistDataHandler,
+        wishlistData,
+        setWishlistData,
+        // toastState,
```
Comment (abhisheksahani-nova) [inline src/context/wishlist-context.jsx]: Remove comments if code doesnt needed

**Review by abhisheksahani-nova — APPROVED** [review]
Clean and readable code , seems good to merge

File: src/context/wishlist-context.jsx (hunk 1 above)
Comment (mikesch803) [reply src/context/wishlist-context.jsx]: sure

**Pull request merged**